juniper/src/http/mod.rs

```diff
@@ -217,6 +217,125 @@ where
     }
 }
 
+/// The Expected structure of the decoded JSON document for either POST or GET requests
```

Comment:
There is nothing JSON-specific about this, please remove.

Reply:
https://github.com/graphql-rust/juniper/pull/608